week2/homework/src/todo.js

```diff
@@ -0,0 +1,76 @@
+const fs = require('fs');
+
+const loadJSON = () => {
+  try {
+    const dataBuffer = fs.readFileSync('todos.json');
+    const dataJSON = dataBuffer.toString();
+    return JSON.parse(dataJSON);
+  } catch (e) {
+    return [];
+  }
+};
+
+const saveJSON = todos => {
+  const dataJSON = JSON.stringify(todos);
+  fs.writeFileSync('todos.json', dataJSON);
+};
+
+const createOrderValue = array => {

```

Comment:
This is correct but it feels like an overkill.
If I wanted the element with order 5 from a javascript array I would use `array[5-1]`
The only possible use for order is when printing the list to the user